packages/dom/test/unit/lib/ElementAssertion.test.tsx

```diff
@@ -42,4 +53,81 @@ describe("[Unit] ElementAssertion.test.ts", () => {
       });
     });
   });
+
+  describe(".toContainElement", () => {
+    context("when the ancestor element contains the descendant element", () => {
+      it("returns the assertion instance", () => {
+        const { getByTestId } = render(<ContainsTestComponent/>);
+
+        const grandparent = getByTestId("grandparent");
+        const parent = getByTestId("parent");
+        const child = getByTestId("child");
+        const svgElement = getByTestId("svg-element");
+
+        const grandparentTest = new ElementAssertion(grandparent);
+        const parentTest = new ElementAssertion(parent);
+
+        expect(grandparentTest.toContainElement(parent));
+        expect(grandparentTest.toContainElement(child));
+        expect(grandparentTest.toContainElement(svgElement));
+        expect(parentTest.toContainElement(child));
+
+        expect(() => grandparentTest.not.toContainElement(parent))
```

Comment:
I would prefer that the inverted assertion next to the normal assertion, just to be clearer of the structure of the test.

Reply:
hi @fonsiher, thank you for the comment. I have completely updated the tests following Jose's suggestions. Please, lmk if they still need updating.